okta/requestExecutor.go

```diff
@@ -114,7 +123,7 @@ func newResponse(r *http.Response) *Response {
 
 func CheckResponseForError(resp *http.Response) error {
 	statusCode := resp.StatusCode
-	if statusCode >= 200 && statusCode <= 299 {
+	if statusCode >= http.StatusOK && statusCode < http.StatusMultipleChoices {
```

Comment:
I see that we're also doing this type of check in the Node SDK, but I'm thinking it's wrong.  I think it should be >=200 < 400, since 3xx codes technically are not errors.

Reply:
changed to `<400`